src/rebar_cmd.app.src

```diff
@@ -1,8 +1,8 @@
 {application, rebar_cmd,
- [{description, "Run custom shell commands as rebar3 tasks"},
-  {vsn, "0.2.6"},
+ [{description, "Run custom shell commands with rebar3 cmd <command>"},
+  {vsn, git},
```

Comment:
I can revert this, but this stops you from having to manually remember this at every release.